src/components/StaleDataIndicator.test.js

```diff
@@ -46,7 +55,7 @@ describe("staleness messaging", () => {
     });
 
     it("shows date if older than yesterday", async () => {
-        const minutesStaleTimestamp = new Date("2021-01-21T00:00");
+        const minutesStaleTimestamp = new Date("2021-01-10T00:00");
```

Comment:
Why? lol

Reply:
Oh this was an artifact from me testing out a mocked Date via `jest.spyOn`. I can revert it if you want